app/controllers/foreman_inventory_upload/uploads_settings_controller.rb

```diff
@@ -0,0 +1,26 @@
+module ForemanInventoryUpload
+  class UploadsSettingsController < ::ApplicationController
+    def index
+      render json: {
+        autoUploadEnabled: Setting[:allow_auto_inventory_upload],
+        hostObfuscationEnabled: Setting[:obfuscate_inventory_hostnames],
+        ipsObfuscationEnabled: Setting[:obfuscate_inventory_ips],
+        excludePackagesEnabled: Setting[:exclude_installed_packages],
+        CloudConnectorStatus: ForemanInventoryUpload::UploadsSettingsController.cloud_connector_status,
```

Comment:
IMO we should get `CloudConnectorStatus` from the polling in accounts_controller as it should keep updating, unlike the rest of the list here which could be retrieved once

cc @ShimShtein

Reply:
We get it from the account_controller, I just thought it would make sense to put it here as well. I can remove it